winml/test/common/SqueezeNetValidator.cpp

```diff
@@ -261,7 +282,7 @@ void ModelValidator::SqueezeNet(
           std::stringstream ss;
           ss << "EXPECTED: " << outDataExpected.GetAt(i) << " , ACTUAL: " << outDataActual.GetAt(i)
                 << "instance " << instance << ", element " << i;
-          WINML_LOG_ERROR(ss.str().c_str());
+          throw winrt::hresult_invalid_argument(std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(ss.str()));
```

Comment:
>invalid_argument [](start = 31, length = 16)

i believe that this is "unexpected"

Reply:
same in other places.

---
In reply to: [370428403](https://github.com/microsoft/onnxruntime/pull/2902#discussion_r370428403) [](ancestors = 370428403)